### ISSUE TITLE
dracut issue on fedora 

### ISSUE THREAD

**Issue by leifliddy** [AsahiLinux/asahi-scripts#15]
I'm testing out that dracut PR that was recently merged https://github.com/AsahiLinux/asahi-scripts/pull/9

It's failing on this ```( cd /; cpio -i < "$VENDORFW/firmware.cpio" )```
It seems like it can't extract the contents on `firmware.cpio` to `/` (within the initramfs env)

```
Jul 14 02:00:00 fedora dracut-pre-udev[354]: :: Asahi: Triggering early load of NVMe modules...
Jul 14 02:00:00 fedora dracut-pre-udev[354]: :: Asahi: Unpacking vendor firmware into initramfs...
Jul 14 02:00:00 fedora dracut-pre-udev[362]: Mounted System ESP /dev/nvme0n1p4 at /run/.system-efi
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw not created: newer or same age version exists
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm: Cannot mkdir: No such file or directory
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm/brcmfmac4355c1-pcie.apple,hawaii-YSBC-m-2.3.txt: Cannot open: No such file or directory
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm/brcmfmac4355c1-pcie.apple,hawaii-YSBC-m-2.5.txt: Cannot open: No such file or directory
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm/brcmfmac4355c1-pcie.apple,hawaii-YSBC-u-4.1.txt: Cannot open: No such file or directory
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm/brcmfmac4355c1-pcie.apple,hawaii-YSBC-u-4.3.txt: Cannot open: No such file or directory
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm/brcmfmac4355c1-pcie.apple,hawaii.bin: Cannot open: No such file or directory
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm/brcmfmac4355c1-pcie.apple,hawaii.clm_blob: Cannot open: No such file or directory
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm/brcmfmac4355c1-pcie.apple,hawaii.txcap_blob: Cannot open: No such file or directory
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm/brcmfmac4364b2-pcie.apple,ekans-HRPN-m-7.5.txt: Cannot open: No such file or directory
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm/brcmfmac4364b2-pcie.apple,ekans-HRPN-m-7.7.txt: Cannot open: No such file or directory
Jul 14 02:00:00 fedora dracut-pre-udev[373]: cpio: vendorfw/brcm/brcmfmac4364b2-pcie.apple,ekans-HRPN-u.txt: Cannot open: No such file or directory
...
```

I'll continue to troubleshoot this and see what I can find. Just wondering whether anyone has experienced this before.

**Comment by leifliddy** [comment]
Update:
I got it to work by commenting out this line in ```module-setup.sh```
```
#ln_r "/lib/firmware/vendor" "/vendorfw"
```

**Comment by marcan** [comment]
Fixed in f67757c0.
